pkg/util/defaults/defaults_support.go

```diff
@@ -28,6 +29,10 @@ func BaseImage() string {
 	return envOrDefault(baseImage, "KAMEL_BASE_IMAGE", "RELATED_IMAGE_BASE")
 }
 
+func OperatorImage() string {
+	return envOrDefault(fmt.Sprintf("%s:%s", ImageName, Version), "KAMEL_OPERATOR_IMAGE", "KAMEL_K_TEST_OPERATOR_CURRENT_IMAGE")
```

Comment:
It doesn't look good that a test env is leaked to the main codebase, but since it's only for 1.8 then it should be ok.